app/car.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+from math import sqrt
+
+
+@dataclass
+class Car:
+    brand: str
+    fuel_consumption: float
+
+    def total_consumption(self, start: list, end: list) -> float:
+        distance = sqrt((end[0] - start[0]) ** 2 + (end[1] - start[1]) ** 2)
+        return round(2 * distance * self.fuel_consumption / 100, 2)
```

Comment:
The formula for calculating the total fuel consumption seems to assume a round trip (multiplying the distance by 2). If this is intentional, it's fine. Otherwise, if you only need the one-way consumption, you should remove the multiplication by 2.